compiler/shared_ast/print.mli

```diff
@@ -125,3 +125,8 @@ module UserFacing : sig
   (** This combines the user-facing value printer and the generic expression
       printer to handle all AST nodes *)
 end
+
+(**/*)
```

Comment:
Coding artefact to clean?

Reply:
Ah no this is in fact an `ocamldoc` directive that tells it to ignore everything that follows from the generated documentation. It allows to keep the proper doc for `skip_wrappers` in `print.mli`, while hiding it to avoid confusion since `Expr.skip_wrappers` should be used instead.